frontend/packages/console-app/src/components/cloud-shell/CloudShellTerminal.tsx

```diff
@@ -28,7 +27,11 @@ type Props = {
 type CloudShellTerminalProps = StateProps & Props;
 
 const CloudShellTerminal: React.FC<CloudShellTerminalProps> = ({ user, onCancel }) => {
-  const [namespace, setNamespace] = React.useState(getCloudShellNamespace());
+  const [namespace, setNamespace] = useUserSettings(
+    CLOUD_SHELL_NAMESPACE_CONFIG,
+    '',
+    CLOUD_SHELL_NAMESPACE,
```

Comment:
Just a suggestion, If we rearrange the second and third positions of the arguments (which has default assignments) in this `useUserSettings` hook, then the caller function signature will be improved.

eg: useUserSetting(CLOUD_SHELL_NAMESPACE_CONFIG, CLOUD_SHELL_NAMESPACE) Third argument will always have empty string if the argument isn't passed. WDYT?

Reply:
Updated to make use of just default value

```
const [topologyLayoutData, setTopologyLayoutData] = useUserSettings(
    TOPOLOGY_LAYOUT_CONFIG_STORAGE_KEY,
    localStorage.getItem(TOPOLOGY_LAYOUT_LOCAL_STORAGE_KEY) || '',
  );
```